R/parallel.R

```diff
@@ -110,49 +150,60 @@ hipercow_parallel_set_cores <- function(cores, envir = NULL) {
 
 
 
-hipercow_parallel_setup <- function(method) {
-  if (is.null(method)) {
+hipercow_parallel_setup <- function(parallel) {
+  if (is.null(parallel$method)) {
     return()
   }
 
-  cores <- hipercow_parallel_get_cores()
-  if (is.na(cores)) {
+  processes <- hipercow_parallel_get_cores() %/% parallel$cores_per_process
```

Comment:
Should we be printing something here to the logs if this is wasteful?

Reply:
Added a cli_alert_info